task_11/task_11.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" type="text/css">
+</head>
+<body class="body">
+<header class="header"> 
+    <div class="main_header">
+        <div class="cell">Name</div>
+        <div class="cell">Owner</div>
+        <div class="actions">Actions</div>
+  </div>
+</header>
+
+<main style="margin-top: 50px;">
+
+<div class="main">
+    <div class="cell">Ace</div>
+    <div class="cell">Belarus</div>
+    <div class="buttons">
+      <div class="edit">Edit</div>
+      <div class="edit deleted">Deleted</div>
+    </div>
+</div>
+
+<div class="main">
+    <div class="cell">Align</div>
+    <div class="cell">Belarus</div>
+    <div class="buttons">
+      <div class="edit">Edit</div>
+      <div class="edit deleted">Deleted</div>
+    </div>
+</div>
+
+<div class="main" >
+    <div class="cell">Ambi Pur</div>
+    <div class="cell">Russia</div>
+    <div class="buttons">
+      <div class="edit">Edit</div>
+      <div class="edit deleted">Deleted</div>
+    </div>
+</div>
+
+<div class="main" >
+    <div class="cell">Ariel</div>
+    <div class="cell">USA</div>
+    <div class="buttons">
+      <div class="edit">Edit</div>
+      <div class="edit deleted">Deleted</div>
```

Comment:
"edit" then "delete"
Not understandable

You should have created the common class for button and the modifiers difference.